packages/relay/src/routes/postRoute.ts

```diff
@@ -57,6 +61,33 @@ export default (
         })
     )
 
+    app.get('/api/post/postHistory', async (req, res) => {
+        try {
+            const fromEpoch = parseInt(req.query.from_epoch as string)
+            const toEpoch = parseInt(req.query.to_epoch as string)
+            if (isNaN(fromEpoch) || isNaN(toEpoch)) {
+                throw InvalidParametersError
+            }
+
+            const history = await postService.getPostHistory(
+                fromEpoch,
+                toEpoch,
+                db
+            )
+            res.status(200).json(history)
+        } catch (error) {
```

Comment:
Same as comment. Any benefit to handle the error in try catch here. Instead of throwing the error directly?

Reply:
fixed